corehq/apps/hqwebapp/static/hqwebapp/js/lib/jquery-ui/multiselect/ui.multiselect.js

```diff
@@ -138,6 +138,18 @@ $.widget("ui.multiselect", {
 
 		$.widget.prototype.destroy.apply(this, arguments);
 	},
+	refresh: function() {
+		this.container.remove();
+		this._create();
+	},
+	addSelected: function(value, text) {
+		this.element.append(
+			$("<option/>").prop('value', value)
+			              .prop('selected', 'selected')
+			              .text(text)
+		);
+		this.refresh();
+	},
```

Comment:
I'm not sure - I guess it depends on what exactly we want it for.  We currently use this widget in several places, so if we switched one we should switch them all.  This widget is nice because it gives you a full-page side-by-side view, whereas most widgets are based on a text box dropdown.  The latter seems inappropriate when the select action is the primary function of the page.

I imagine @biyeun has some ideas about how we should be doing this.  The nice thing is that these are pretty plug-n-play (with the exception of this particular instance, where I needed to dynamically insert into the widget), so changing wouldn't be too hard.